app/containers/message/Attachments.tsx

```diff
@@ -29,42 +35,55 @@ const AttachedActions = ({ attachment, theme }: IMessageAttachedActions) => {
 };
 
 const Attachments = React.memo(
-	({ attachments, timeFormat, showAttachment, getCustomEmoji, theme }: IMessageAttachments) => {
+	({ attachments, timeFormat, showAttachment, style, getCustomEmoji, isReply }: IMessageAttachments) => {
 		if (!attachments || attachments.length === 0) {
 			return null;
 		}
 
-		return attachments.map((file: any, index: number) => {
-			if (file.image_url) {
+		if (attachments.length === 2 && attachments[0].image_url && attachments[1].message_link) {
+			attachments = [attachments[1], attachments[0]];
+		}
```

Comment:
This looks ugly but is the only way I could think of in order to achieve `Image` to be after `Reply` 🙈 
<img width="311" alt="image" src="https://user-images.githubusercontent.com/36211892/158936658-a5de26b2-a1ce-4ccb-a566-fac1ba2153e8.png">